diesel/src/associations/belongs_to.rs

```diff
@@ -14,7 +14,12 @@ pub trait BelongsTo<Parent> {
     fn foreign_key(&self) -> Option<&Self::ForeignKey>;
     fn foreign_key_column() -> Self::ForeignKeyColumn;
 }
-
+/// `grouped_by` takes a `Vec<Child>` and a `&Vec<Parent>` and returns a `Vec<Vec<Child>>`
```

Comment:
Reading this from the perspective of a new user -- What is `Child`? What is `Parent`? Why does this say it takes two arguments, but the signature only takes one?